### PR TITLE
🌱Add manifest collection for e2e test

### PR DESCRIPTION
**Pull request by smoshiur1237**
It will add manifest collection for e2e test.

### CODE REVIEW

File: scripts/fetch_manifests.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  ipclaim
+  ipaddress
+  m3data
+  m3dataclaim
+  m3datatemplate
+)
+
+NAMESPACES="$(kubectl get namespace -o jsonpath='{.items[*].metadata.name}')"
+for NAMESPACE in ${NAMESPACES}
+do
```
Comment (tuminoid) [inline scripts/fetch_manifests.sh:34]: Please put `do` on the same line as for.

File: scripts/fetch_manifests.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  for kind in "${manifests[@]}"; do
+    mkdir -p "${DIR_NAME}"/"$kind"
```
Comment (tuminoid) [inline scripts/fetch_manifests.sh:36]: `${DIR_NAME}/${kind}" simplifies quotes and adds curlys.

File: scripts/fetch_manifests.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    for name in $(kubectl get -n "${NAMESPACE}" -o name "$kind" || true)
+    do
```
Comment (tuminoid) [inline scripts/fetch_manifests.sh:38]: Please put `do` on the same line as for.

File: scripts/fetch_manifests.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  m3dataclaim
+  m3datatemplate
+)
+
+NAMESPACES="$(kubectl get namespace -o jsonpath='{.items[*].metadata.name}')"
+for NAMESPACE in ${NAMESPACES}
+do
+  for kind in "${manifests[@]}"; do
+    mkdir -p "${DIR_NAME}"/"$kind"
+    for name in $(kubectl get -n "${NAMESPACE}" -o name "$kind" || true)
```
Comment (tuminoid) [inline scripts/fetch_manifests.sh:37]: Curlys

File: scripts/fetch_manifests.sh [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    do
+      kubectl get -n "${NAMESPACE}" -o yaml "$name" | tee "${DIR_NAME}"/"$kind"/"$(basename "$name")".yaml || true
```
Comment (tuminoid) [inline scripts/fetch_manifests.sh:39]: Curlys, and also simplify the quotes

**Review by tuminoid — CHANGES_REQUESTED** [review]

**Review by lentzi90 — COMMENTED** [review]
/lgtm
Putting hold also to make sure we can check that the manifests are there after the tests finish.
/hold

**Review by tuminoid — APPROVED** [review]
/lgtm

File: test/e2e/pivoting.go [hunk 6]
```diff
@@ -83,6 +83,22 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 		fetchContainerLogs(&ironicContainers, input.ArtifactFolder, input.E2EConfig.GetVariable("CONTAINER_RUNTIME"))
 	}
 
+	By("Fetch manifest for ephemeral cluster")
+	path := filepath.Join(os.Getenv("CAPM3PATH"), "scripts")
+	cmd := exec.Command("./fetch_manifests.sh") // #nosec G204:gosec
+	cmd.Dir = path
+	outputPipe, _ := cmd.StdoutPipe()
+	errorPipe, _ := cmd.StderrPipe()
```
Comment (mboukhalfa) [inline test/e2e/pivoting.go:91]: would prefer to use `cmd.CombinedOutput()`

Comment (smoshiur1237) [reply test/e2e/pivoting.go:91]: We have done similar in other command run in e2e. So we can improve them in future.

**Comment by smoshiur1237** [comment]
The manifest collection can be checked for e2e : https://jenkins.nordix.org/job/metal3_capm3_e2e_integration_test_centos/105/
It has the deployment manifest and other manifests.

**Pull request merged**